src/FS.Physical/PhysicalFileProvider.cs

```diff
@@ -64,26 +64,115 @@ private PhysicalFileProvider(string root, PhysicalFilesWatcher physicalFilesWatc
                 throw new DirectoryNotFoundException(Root);
             }
 
-            _filesWatcher = physicalFilesWatcher;
             _filters = filters;
+            _fileWatcherFactory = () => CreateFileWatcher();
+        }
+
+        /// <summary>
+        /// Gets or sets a value that determines if this instance of <see cref="PhysicalFileProvider"/>
+        /// uses polling to determine file changes.
+        /// <para>
+        /// By default, <see cref="PhysicalFileProvider"/>  uses <see cref="FileSystemWatcher"/> to listen to file change events
+        /// for <see cref="Watch(string)"/>. <see cref="FileSystemWatcher"/> is ineffective in some scenarios such as mounted drives.
+        /// Polling is required to effectively watch for file changes.
+        /// </para>
+        /// <seealso cref="UseActivePolling"/>.
+        /// </summary>
+        /// <value>
+        /// Defaults to <see langword="true"/>, if the environment variable <c>DOTNET_USE_POLLING_FILE_WATCHER</c> is <c>true</c> or <c>1</c>.
```

Comment:
Please change the wording so it's clear that the default is actually `false`, and it will take whatever value in the ENV otherwise.